Sources/Apodini/Semantic Model Builder/gRPC/GRPCService/GRPCServiceUnary.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  GRPCServiceUnary.swift
+//  
+//
+//  Created by Moritz Schüll on 20.12.20.
+//
+
+import Foundation
+@_implementationOnly import Vapor
+
+// MARK: Unary request handler
+extension GRPCService {
+    func createUnaryHandler<C: Component>(for component: C, with context: Context) -> (GRPCRequest) -> EventLoopFuture<C.Response> {
+        { (request: GRPCRequest) in
+            let guardEventLoopFutures = self.processGuards(request, with: context)
+            return EventLoopFuture<Void>
+                .whenAllSucceed(guardEventLoopFutures, on: request.eventLoop)
+                .flatMap { _ in
+                    request.enterRequestContext(with: component) { component in
+                        var response: Encodable = component.handle()
+                        for responseTransformer in context.get(valueFor: ResponseContextKey.self) {
+                            response = request.enterRequestContext(with: responseTransformer()) { responseTransformer in
+                                responseTransformer.transform(response: response)
+                            }
+                        }
+                        // swiftlint:disable force_cast
+                        return request.eventLoop.makeSucceededFuture(response as! C.Response)
```

Comment:
I'm not sure if we can cast this to C.Response here? It might be possible that response transformers modify the return type. 🤔